packages/features/ee/payments/api/webhook.ts

```diff
@@ -155,7 +162,11 @@ async function handlePaymentSuccess(event: Stripe.Event) {
 
   await prisma.$transaction([paymentUpdate, bookingUpdate]);
 
-  await sendScheduledEmails({ ...evt });
+  if (!isConfirmed && !eventTypeRaw?.requiresConfirmation) {
+    await handleConfirmation({ user, evt, prisma, bookingId: booking.id, booking, paid: true });
+  } else {
+    await sendScheduledEmails({ ...evt });
```

Comment:
How is the case where isConfirmed is false and event requires confirmation? `sendScheduledEmails` seem to send confirmed booking emails only.

Reply:
If the booking is not already confirmed and doesn't require confirmation. I didn't want to modify too much the current logic